example/src/screens/NetworkClientScreen.tsx

```diff
@@ -161,21 +161,25 @@ export default function NetworkClientScreen({navigation, route}) {
     const makeRequest = async () => {
         const options = {
             headers: sanitizeHeaders(requestHeaders),
-            body: JSON.parse(body),
         };
-        let requestMethod;
-        switch (method.toLowerCase().trim()) {
-            case 'get':
-                requestMethod = client.get;
-                break;
-            case 'post':
-                requestMethod = client.post;
-                break;
-            default:
-                break;
+        const methodName = method.toLowerCase().trim();
```

Comment:
non blocking, but here you are trimming the method name and applying a lowercase to compare, then on  line 374 is totally different.